### DISCUSSION TITLE
Magick -thumbnail creating 2 thumbnails

### DISCUSSION THREAD

**Discussion by chelle-rr** [ImageMagick/ImageMagick#7333] [Help] [answered]
### ImageMagick version

7.1.1-32

### Operating system, version and so on

macOS Ventura 13.4

### Description

I'm using imagemagick in a bash script to create thumbnails. Typically, this line produces only one thumbnail:

`magick "$image_file" -thumbnail '200x200' "$thumbnail_filename"`

However, when it encounters certain images, it creates two: thumbnail_filename-0 and thumbnail_filename-1. The one ending in -1 will be slightly bigger. These certain images appear to have been edited; the unedited original will only create 1 thumbnail, as intended.

<img width="296" alt="Screenshot 2024-05-21 at 2 31 12 PM" src="https://github.com/ImageMagick/ImageMagick/assets/62896863/2130a26e-533b-4a7c-ba89-4590a4d48041">

I checked the images that are creating the extra thumbnail w/ mediainfo, but honestly not sure what to look for. Here's an example of the mediainfo for an image that generated 2 thumbnails:

Format: TIFF
File size: 40.9 MiB

Writing application: EPSON Adobe Photoshop 22.5 (Macintosh)
Writing library: Perfection V600

Image 
Format: Raw
Format settings: Big
Width: 2 170 pixels
Height: 3 296 pixels
Color space: RGB
Bit depth: 8 bits
Compression mode: Lossless
Encoded date: 2024:03:25 17:21:21
Density: 2400.000 dpi

Any help would be appreciated ... thanks!

**Comment by snibgo** [comment, marked as answer]
I expect your input TIFF file contains two images.

What does "`magick Film_3_edit.tif info:`" say?

**Reply by chelle-rr** [reply]
IT IS TWO IMAGES. Weirdly I can't see this in any of the ways I'd usually expect to see multipage TIFFs, so maybe there's more investigating to be done, but this certainly explains why I'm getting two thumbnails. Thanks!!

**Comment by fmw42** [comment]
If you only want one, the first, then add [0] to the end of your file name.  If you want the second, then add [1] to the end of your file name.
